nextcloudappstore/api/v1/release/info.xsd

```diff
@@ -15,7 +15,7 @@
                 <xs:element name="version" type="semver"
                             minOccurs="1" maxOccurs="1"/>
                 <xs:element name="licence" type="licence" minOccurs="1"
-                            maxOccurs="1"/>
+                            maxOccurs="unbounded"/>
```

Comment:
Ah. Both is there. All good. Was just confused because it's on one line in the other cases and so I didn't see it :)